DocumentDataAPI/DocumentDataAPI/Data/Algorithms/TfIdfCalculator.cs

```diff
@@ -0,0 +1,28 @@
+using DocumentDataAPI.Models;
+
+namespace DocumentDataAPI.Data.Algorithms;
+
+//Not in use
+public class TfIdfCalculator
+{
+    private readonly int _docCount;
+    private readonly IEnumerable<WordRatioModel> _wordRatios;
+
+    public TfIdfCalculator(int docCount, IEnumerable<WordRatioModel> wordRatios)
+    {
+        _docCount = docCount;
+        _wordRatios = wordRatios;
+    }
+
+    public double CalculateTfIdf(WordRatioModel wordRatio)
+    {
+        int termdocCount = _wordRatios.Where(x => x.Word == wordRatio.Word).Count();
+
+        return wordRatio.Percent * IdfCalculator(termdocCount);
+    }
+
+    private double IdfCalculator(int termdocCount)
+    {
+        return Math.Log(_docCount / termdocCount);
+    }
+}
```

Comment:
Samme som den anden - find ud af hvordan du kan markere at den ikke er i brug, og henvis til hvad der bliver brugt

Reply:
```suggestion
//Not in use
[Obsolete("Use NpgWordRelevanceRepository to update TF-IDF values in db", false)]
public class TfIdfCalculator
{
    private readonly int _docCount;
    private readonly IEnumerable<WordRatioModel> _wordRatios;

    public TfIdfCalculator(int docCount, IEnumerable<WordRatioModel> wordRatios)
    {
        _docCount = docCount;
        _wordRatios = wordRatios;
    }

    public double CalculateTfIdf(WordRatioModel wordRatio)
    {
        int termdocCount = _wordRatios.Where(x => x.Word == wordRatio.Word).Count();

        return wordRatio.Percent * IdfCalculator(termdocCount);
    }

    private double IdfCalculator(int termdocCount)
    {
        return Math.Log(_docCount / termdocCount);
    }
}
```
Du kan gøre sådan her, det er ret smart, den første parameter er en grund, og den anden er om der skal laves en error frem på warning i compile time